Resources/config/default_settings.yml

```diff
@@ -0,0 +1,6 @@
+parameters:
+    # how many days banner should be hidden when user accepts policy?
+    ez_privacy_cookie.days: 365
+
+    # name to be used to store cookie status
+    ez_privacy_cookie.cookie_name: "EzPrivacyCookieStatus"
```

Comment:
Such a think will force it to require eZ Publish/Platform I think, but @bdunogier can probably comment on that.